mbed/mbed.py

```diff
@@ -1060,6 +1058,12 @@ def fromlib(cls, lib=None):
         m_local = re.match(regex_local_ref, ref.strip().replace('\\', '/'))
         m_repo_ref = re.match(regex_url_ref, ref.strip().replace('\\', '/'))
         m_bld_ref = re.match(regex_build_url, ref.strip().replace('\\', '/'))
+
+        if m_repo_ref:
+            rev = m_repo_ref.group(3)
+            if rev and not re.match(r'^([a-fA-F0-9]{6,40})$', rev):
+                error('named branches not allowed in .lib, offending lib is {} '.format(os.path.basename(lib)))
```

Comment:
Could you please capitalize first letter in the error. Sorry for nitpicking - just for consistency.

Reply:
done.